banyand/series/trace/write.go

```diff
@@ -55,13 +54,16 @@ func (t *traceSeries) Write(seriesID common.SeriesID, shardID uint, entity data.
 	seriesIDBytes := convert.Uint64ToBytes(intSeriesID)
 	wp := t.writePoint(ts)
 
-	err = wp.TimeSeriesWriter(shardID, dataStoreName).Put(seriesIDBytes, entityVal.DataBinaryBytes(), ts)
+	err = wp.TimeSeriesWriter(shardID, dataStoreName).Put(seriesIDBytes, entityVal.GetDataBinary(), ts)
 	if err != nil {
-		return 0, errors.Wrap(err, "failed to write traceSeries data")
+		return 0, errors.Wrap(err, "fail to write traceSeries data")
 	}
-	builder := flatbuffers.NewBuilder(0)
-	builder.Finish(fb.CopyFields(entityVal, builder))
-	err = wp.TimeSeriesWriter(shardID, fieldsStoreName).Put(seriesIDBytes, builder.FinishedBytes(), ts)
+
+	byteVal, err := proto.Marshal(entityVal)
```

Comment:
entityVal should write into fieldsStore without databinary. Could you extract fields from entityVal,  then save them into the below kv store?

Reply:
Fixed